### ISSUE TITLE
setData() not working with <Select />

### ISSUE THREAD

**Issue by zys** [italoiz/unform-community-packages#17]
The `<Select />` component does not work with data sent from an api.

```
import React, { useRef, useEffect } from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Grid from '@material-ui/core/Grid';
import { TextField, Select } from 'unform-material-ui';
import MenuItem from '@material-ui/core/MenuItem';
import { Form } from '@unform/web';
import { Scope } from '@unform/core';

import api from '../../services/api';

const styles = {
  gridRow: {
    margin: 0,
  },
  fullWidth: {
    width: '100%',
  },
};

const useStyles = makeStyles(styles);

export default function Example(props) {
  const classes = useStyles();
  const formRef = useRef(null);

  useEffect(() => {
    async function fetchData() {
      const dat = (await api.get(`someRoute/${props.match.params.id}`)).data;
      formRef.current.setData(dat);
    }

    fetchData();
  });

  return (
    <Form ref={formRef}>
      <Grid container item spacing={3} xs={12} className={classes.gridRow}>
        <Grid item xs={12} md={6}>
          <TextField name="name" label="Name" fullWidth />;
        </Grid>
        <Scope path="endereco">
          <Grid item xs={12} md={6}>
            <Select
              className={classes.fullWidth}
              name="country"
              label="Country"
            >
              <MenuItem value="br">Brazil</MenuItem>
              <MenuItem value="us">United State</MenuItem>
            </Select>
          </Grid>
        </Scope>
      </Grid>
    </Form>
  );
}
```

Console outputs this error:
SelectInput.js:301 Material-UI: you have provided an out-of-range value undefined for the select (name="country") component.
Consider providing a value that matches one of the available options or ''.
The available values are Brazil, United States.

This happens even with the correct value being sent, regardless of "Brazil" or "United States".

**Comment by naldo-88** [comment]
Hello,
I'm also having a hard time.
Would you help me.
![code](https://user-images.githubusercontent.com/65252831/82086117-6629e700-96c4-11ea-9646-99cd36b554a6.png)

**Comment by italoiz** [comment]
This bug is resolved in the next release **v1.0.1** of [unform-material-ui](https://npmjs.com/package/unform-material-ui)
